### PR TITLE
SPM-991: Fix latest_package_cache view

### PR DESCRIPTION
**Pull request by AlesKas**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

**Review by Josca — CHANGES_REQUESTED** [review]
some conflict appeared, please fix

File: database_admin/migrations/066_package_latest_redefine.down.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+DROP FUNCTION IF EXISTS refresh_latest_packages_view;
+DROP INDEX IF EXISTS package_latest_cache_pkey;
+DROP MATERIALIZED VIEW IF EXISTS package_latest_cache;
```
Comment (semtexzv) [inline database_admin/migrations/066_package_latest_redefine.down.sql:3]: If we run the down migration, it'll delete the view and the refresh task will fail. I'd leave the down migration empty

**Review by semtexzv — APPROVED** [review]

**Review by Josca — APPROVED** [review]
Looks good, thanks :+1:

**Pull request merged**